### DISCUSSION TITLE
Trigger  like update

### DISCUSSION THREAD

**Discussion by lazyclaw-labs** [linq2db/linq2db#3176] [General]
let's say I have a column: UpdateTime, which should be udpated to the current time whenever whatever columns got updated.

In Linq2DB, is there anyway to make this automatic?

**Comment by MaceWindu** [comment]
Not sure I understand what you mean here.

`whenever whatever columns got updated`:
- if you mean update in database - than this is a task for database trigger
- if by than you mean update by linq2db, than:
 
1. in general NO as there are a lot of ways to do it: Insert as object API, Insert as LINQ expression API, raw SQL, BulkCopy, Merge...
2. for LINQ queries IIRC you can add query prerpocessing, check this discussion for starters #2142
